inference/core/interfaces/http/http_api.py

```diff
@@ -612,6 +613,44 @@ async def clip_compare(
                         trackUsage(clip_model_id, actor, n=2)
                     return response
 
+            if CORE_MODEL_DOCTR_ENABLED:
+                @app.post(
+                    "/doctr/ocr",
+                    response_model=M.DoctrOCRInferenceResponse,
```

Comment:
`data_models as M` is no longer imported in this file. You'll need to add imports for each request and response object individually from `inference/core/entities`